_posts/2022-05-23-release-kubernetes-operator-1.0.0.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title:  "Apache Flink Kubernetes Operator 1.0.0 Release Announcement"
+subtitle: "Lifecycle management for Apache Flink deployments using native Kubernetes tooling"
+date: 2022-05-23T08:00:00.000Z
+categories: news
+authors:
+- gyfora:
```

Comment:
Please add yourself as author @wangyang0918

Reply:
@gyfora Since the blog post is authored by you, I think it is ok to have your name here :)